flixel/util/FlxGradient.hx

```diff
@@ -1,14 +1,14 @@
 package flixel.util;
 
-import flash.display.Bitmap;
```

Comment:
I was getting compiler errors. Might have just been because I was using some WIP commits from OpenFL at the time.